ProjectSettings/QualitySettings.asset

```diff
@@ -4,7 +4,7 @@
 QualitySettings:
   m_ObjectHideFlags: 0
   serializedVersion: 5
-  m_CurrentQuality: 0
+  m_CurrentQuality: 5
```

Comment:
Did you intend to make this project settings change? It seems out of place in this PR.